x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -341,7 +356,17 @@ export const processGroupByResults = (
     const documentCount = group.documentCount;
 
     if (checkValueAgainstComparatorMap[count.comparator](documentCount, count.value)) {
-      const alertInstance = alertInstanceFactory(group.name, count.value, documentCount);
+      const alertInstance = alertInstanceFactory(
+        group.name,
+        getReasonMessageForGroupedCountAlert(
+          documentCount,
```

Comment:
Just a simple comment nothing really wrong about that. `getReasonMessageForGroupedCountAlert` accepts as  first parameter the actual value and second parameter the expected value. The `alertInstanceFactory` accepts first the expected value and then the actual value. Maybe a symmetry in the order of parameters between the two functions would makes sense? I just had to ensure that arguments were passed the correct way because they were the other way around.

Reply:
Good point - I'll change the order of arguments.

Reply:
:information_source: I merged without it to unblock dependent PRs. I'll create a separate PR for the improvement.